.github/workflows/test-on-ryot.yml

```diff
@@ -0,0 +1,105 @@
+---
+# Run 'compile_and_test_for_board.py' on connected boards.
+#
+# This workflow will run on a RYOT (Run Your Own Test) machine and launch
+# all tests on all boards currently connected to that machine. This
+# workflow relies on self-hosted runners, this means building and
+# flashing happens on the self-hosted runner (all in docker). An alternative
+# approach was used in https://github.com/RIOT-OS/RIOT/pull/14600, where
+# the RYOT machine was only used for remote flashing, this is a better
+# alternative if ssh access is possible.
+#
+# Documentation:
+#
+# * Setup a RYOT machine:
+#   https://github.com/fjmolinas/riot-ryot/blob/master/setup.md
+#
+# * Requirements (already filled by a RYOT machine):
+#   * Add one or more self-hosted runners:
+#     https://docs.github.com/en/actions/hosting-your-own-runners/adding-self-hosted-runners
+#   * All required flashing tools installed
+#   * udev rules that map all BOARD to /dev/riot/tty-$(BOARD), see
+#     http://riot-os.org/api/advanced-build-system-tricks.html#multiple-boards-udev
+#   * RIOT_MAKEFILES_GLOBAL_PRE that sets PORT and DEBUG_ADAPTER_ID for each
+#     BOARD
+#   * A list of connected BOARDs in JSON format so that fromJSON can be used
+#     to dynamically setup the matrix, e.g. of a make target providing this:
+#       - make target example: https://github.com/fjmolinas/riot-ryot/blob/72fc9ad710a2219e942c5965a014e934822e9da5/template/conf/makefiles.pre#L19-L24
```

Comment:
The bullet point list and specifically referring to the make target example is now redundant:

```suggestion
#     to dynamically setup the matrix, e.g. of a make target providing this. See:
#     https://github.com/fjmolinas/riot-ryot/blob/72fc9ad710a2219e942c5965a014e934822e9da5/template/conf/makefiles.pre#L19-L24
```

Reply:
Would:

```
#   * A list of connected BOARDs in JSON format so that fromJSON can be used
#     to dynamically setup the matrix, e.g. make target providing this:
#     https://github.com/fjmolinas/riot-ryot/blob/72fc9ad710a2219e942c5965a014e934822e9da5/template/conf/makefiles.pre#L19-L24
```

be ok as well?

Reply:
Sure.

Reply:
done